openlibrary/templates/publishers/view.html

```diff
@@ -157,7 +157,7 @@ <h3>$_("Authors")</h3>
         <span class="count">$sprintf(ungettext("%s edition", "%s editions", p.count), commify(p.count))</span>
          <br/>
 
-$if "lists" in ctx.features:
+$if "lists" not in ctx.features:
```

Comment:
This does not quite work how you possibly think it does. ctx.features is set by site admin.
So if side admin turns off the list feature (which currently defaults to on) it would show up on this page.

I think if you want to remove the lists from the page that instead you want to remove the entire if statement and its body.

Reply:
I was ambiguous with ctx.features and how it is going to be used(set). Thanks for clearing things up. 
Yes, removing the entire body of the **IF-structure**(#3197 :v3) did the trick.
Let me know if the issue is resolved or not.